### PR TITLE
MDCT-355: Remove CMS User note

### PR DESCRIPTION
**Pull request by ntsummers1**
# Description

Fixes [MDCT-355](https://qmacbis.atlassian.net/browse/MDCT-355)

Hated seeing the CMS User note? So did the rest of the world. It's done, kapeesh? It won't be bothering us no more. 

Screen Shot 2022-06-22 at 11 17 19 AM

## How to test

./dev local
Log in
See that the main page is filled with only the most effective wording now known to man.

## Dependencies

# Get it done

Now that the PR is open this is what happens next

## Code authors checklist

- [x] I have performed a self-review of my code
- [x] I have added thorough tests. [Testing Doc](https://qmacbis.atlassian.net/wiki/spaces/CM/pages/2914025525/Test+Suite+and+Testing+Research)
- [x] Necessary analytics were added, or no new analytics were required
- [x] I have made corresponding changes to the documentation
- [x] I have assigned the PR to myself

## Reviewers Checklist (Two different people)

- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

## Assignee

- [ ] I have closed the PR after the review and necessary changes (squashing preferred)

### CODE REVIEW

File: services/ui-src/src/components/sections/homepage/CMSHomepage.js [hunk 1]
```diff
@@ -112,16 +112,11 @@ const CMSHomepage = ({
...
       <div className="ds-l-container-large">
         {currentUserRole !== UserRoles.BUSINESS_OWNER_REP ? (
-          <>
-            <div className="ds-l-row ds-u-padding-left--2">
-              <h1 className="page-title ds-u-margin-bottom--0">
-                CHIP Annual Reporting Template System (CARTS)
-              </h1>
-            </div>
-            <div className="page-info ds-u-padding-left--2">
-              <div className="edit-info">CMS user</div>
```
Comment (BearHanded) [inline services/ui-src/src/components/sections/homepage/CMSHomepage.js:122]: There's a blurb like this on the other homepages as well for their relevant role. They get modified by my current PR as well. Do they need to get pulled out? They say Help Desk & CMS Admin

Comment (ntsummers1) [reply services/ui-src/src/components/sections/homepage/CMSHomepage.js]: Per David its allllllll got to go. Update now made.

**Review by BearHanded — CHANGES_REQUESTED** [review]

**Review by BearHanded — APPROVED** [review]
Thanks for the quick update!

**Review by gmrabian — APPROVED** [review]
looks good

**Pull request merged**